### PR TITLE
update image url for shagunsodhani as the previous url was not workin…

### PR DESCRIPTION
**Pull request by shagunsodhani**
…g on the mobile app

### CODE REVIEW

**Comment by CuriousLearner** [comment]
@shivan1b Can you please check if this would work for the mobile app?

File: api/tracks.json [hunk 1]
```diff
@@ -237,7 +237,7 @@
 			"speaker": {
 				"name": "Shagun Sodhani",
 				"info": "I am a developer working with the Analytics and Data Science team at Adobe Systems where I contribute to solving novel problems in the domain of Machine Learning. I recently won the Outstanding Young Engineers Award as well. I have been actively using Python for the past 4 years. I have previously given talks at PyCon India and Big Data Training Program, IIT Roorkee. I also worked as a teaching assistant for the 3-course series on Data Science and Engineering with Spark XSeries, created by professors from University of California, Berkeley, University of California, Los Angeles and Databricks and offered on the edX platform.",
-				"photo": "https://paste.opensuse.org/images/70594065.png",
+				"photo": "https://dl.dropboxusercontent.com/u/56860240/profile.png",
```
Comment (CuriousLearner) [inline api/tracks.json:240]: I guess the problem was with SSL certificates, but let's wait for someone from the mobile app team to have a look at this.

**Comment by akshayaurora** [comment]
@shagunsodhani thanks for the work on this but I just uploaded your photo to paste.opensuses.org. That works.

**Comment by shagunsodhani** [comment]
@akshayaurora ouch. I did not realize that. Thanks for taking care of this. :+1:

**Pull request closed without merging**